python_modules/dagster/dagster/generate/new_project/new_project/graphs/my_graph.py.tmpl

```diff
@@ -0,0 +1,13 @@
+from dagster import graph
+
+from {{ repo_name }}.ops.hello import hello
+
+@graph
+def my_graph():
```

Comment:
If we can come up with a good name, it would be preferable to avoid the word "graph" inside the graph name.  That's because the graph name will by default become the job name, and having a job named "my_graph" could be confusing.

Maybe something like `do_it_all` or `say_hello`?